src/lib.rs

```diff
@@ -1268,9 +1268,9 @@ impl Rtc {
                 }
                 SctpEvent::Close { id } => {
                     let Some(id) = self.chan.channel_id_by_stream_id(id) else {
-                        warn!("Drop ChannelClose event for id: {:?}", id);
```

Comment:
Why does this warning go away?

Reply:
right you are correct that should stay let me fix.